src/BasicEntities/Entity/Book.php

```diff
@@ -35,15 +37,20 @@ class Book implements TranslatableInterface
 
     /**
      * @ORM\Id
+     *
      * @ORM\GeneratedValue
+     *
      * @ORM\Column(type="integer")
      */
     private ?int $id = null;
 
     /**
      * @Assert\Valid
+     *
      * @Gedmo\SortableGroup
+     *
      * @ORM\ManyToOne(targetEntity="Category", inversedBy="books")
+     *
```

Comment:
This spaces should not be applied. Also, could you change those annotations to attributes. Since you dropped php 7.4, attributes can be used as the only format.

This applied for ORM, Gedmo, and asserts. (You might need to update the ORM configuration accordingly)